llpc/lower/llpcSpirvLowerGlobal.h

```diff
@@ -92,6 +96,7 @@ class SpirvLowerGlobal : public SpirvLower, public llvm::InstVisitor<SpirvLowerG
   void interpolateInputElement(unsigned interpLoc, llvm::Value *interpInfo, llvm::CallInst &callInst);
 
   llvm::Value *toInt32Value(llvm::Value *value, llvm::Instruction *insertPos);
+  Value *getConstantInt32(unsigned c);
```

Comment:
Maybe change its name to getInt32 to keep compatibility with IR builder
c -> constant (we don't use single-character as variable name, this is somewhat different from LLVM)

Reply:
I missed IRBuilder has a getInt32() function, I think that is just simple enough to make code short, I will just remove the getConstantInt32().